autotest/gdrivers/zarr_driver.py

```diff
@@ -43,8 +43,10 @@ def module_disable_exceptions():
     gdal.GDT_UInt32: "I",
     gdal.GDT_Int64: "q",
     gdal.GDT_UInt64: "Q",
+    # gdal.GDT_Float16: "e",
```

Comment:
Commented code in that file should be either removed or re-enabled

Reply:
I re-enabled it.